spec/STORE.md

```diff
@@ -71,9 +73,7 @@ Local dependencies are symlinked in a Windows/MacOS/Linux compatible way, not co
 
 A file that contains the store graph. All keys should be sorted.
 
-> Question: What parts of this file are redundant to the information that could be derived by walking the symlink tree? If the store.yaml and filesystem contain conflicting information, how do we resolve the corruption?
-
-> zkochan: the `dependents` part. I'd say the store.yaml is the single place of truth and if in the filesystem something does not match, we can force-update it.
+The store.yaml is the single place of truth. If something in the filesystem does not match the graph described in the store.yaml, the corrupted/incorrect directories of the store are recreated.
```

Comment:
Sure they can be deleted, but should we specify it in the spec? I think package managers can decide whether they want to clean up or not